src/packages/builder/src/codegen/index.ts

```diff
@@ -1,114 +1,113 @@
 import fs from 'fs';
 import path from 'path';
-import { executeCodegen } from '@graphql-codegen/cli';
+import { codegen } from '@graphql-codegen/core';
 import nearOperationFilePreset from '@graphql-codegen/near-operation-file-preset';
+import * as typescriptOperations from '@graphql-codegen/typescript-operations';
+import * as typescriptReactApollo from '@graphql-codegen/typescript-react-apollo';
+import { GraphQLSchema, buildSchema, printSchema, parse, DocumentNode } from 'graphql';
+import * as typescript from '@graphql-codegen/typescript';
+import { loadDocuments } from '@graphql-tools/load';
+import { CodeFileLoader } from '@graphql-tools/code-file-loader';
+import { Source } from '@graphql-tools/utils';
 
 type CodegenOptions = {
 	typesOutputPath?: string | string[];
 };
 
+type CodegenConfig = { [key: string]: any };
```

Comment:
I'd just inline `Record<string, any>` instead of declaring a type for this I think.

Reply:
Simplification applied